core/src/main/java/io/undertow/server/handlers/resource/DirectoryUtils.java

```diff
@@ -90,7 +94,7 @@ public static boolean sendRequestedBlobs(HttpServerExchange exchange) {
         return false;
     }
 
-    public static StringBuilder renderDirectoryListing(String path, Resource resource) {
+    public static StringBuilder renderDirectoryListing(final HttpServerExchange exchange, String path, final Resource resource) {
```

Comment:
@baranowb can the original method be deprecated with an attribute marking it for removal on 2.4.0.Final?

Reply:
@fl4via yes, however, the question is this fix required, given that more or less this can be achieved with some URL rewriting?